scripts/iftop-postprocess.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#
+# Reports network bandwidth usage
+#
+set -e
+
+cd "$(dirname "$0")"
+
+awk '{ if ($3 ~ "=>") { print $2, $7 } else if ($2 ~ "<=") { print $1, $6 }} ' < "$1" \
```

Comment:
roll this awk into `solana-network-tool`?

Reply:
I was hoping to roll in more functionality into the tool. The arguments can decide what to do.

Reply:
I'll look into it. Ok to do it in a follow up PR?

Reply:
follow-up is fine with me